lib/cfe_internal_hub.cf

```diff
@@ -134,6 +134,8 @@ AND    t.$(settings[$(index)][time_key]) <= (z.latest - '$(settings[$(index)][hi
 
   commands:
       "$(sys.bindir)/psql cfdb -c \"$(remove_query_$(settings[$(index)][report]))\""
+        inform => "false",
```

Comment:
ooop, the `inform` attribute to suppress command output while in inform mode was first introduced in 3.15.0.
https://docs.cfengine.com/docs/3.17/reference-promise-types-commands.html#inform

Since 3.12.7 is still supported, we need to maintain backwards compatibility. This needs to be instrumented so that the attribute is not used on versions older than 3.15.0. See prior art inside MPF:

https://github.com/cfengine/masterfiles/blob/ffb795996e63f812c11bf46cfd952d5019553eee/cfe_internal/enterprise/CFE_hub_specific.cf#L586

Reply:
I just pushed to your branch.

Reply:
huh! I wonder how deploytests will feel on 3.12.x. I'll investigate, thanks!